### ISSUE TITLE
Port features from SESE/Essentials

### ISSUE THREAD

**Issue by rexxar-tc** [TorchAPI/Essentials#36]
If you want a feature from SESE/Essentials ported to Torch Essentials, post here. Vote on features you want with reactions. Higher votes = higher priority.

An incomplete list of features is located in the Essentials repo Readme: https://github.com/SEServerExtender/EssentialsPlugin

Comments with 🎉 in the reactions have been implemented!

Note, this is only for features that **already exist in SESE/Essentials.** New feature requests should go in their own issue! Irrelevant or discussion comments will be deleted so I can keep this list clean and easy to read!

**Comment by Neimoh** [comment]
- Triggered cleanups

**Comment by Neimoh** [comment]
- info commands so players can get private messages of saved gps points

**Comment by Neimoh** [comment]
- custom tailored motd's for new or returning members

**Comment by GornyakManiac** [comment]
Server configs editing on the fly (without stopping server)

**Comment by Neimoh** [comment]
- reserved admin slots

**Comment by Gwindalmir** [comment]
Safe Docking

**Comment by GornyakManiac** [comment]
"/utility grids list" analogue to find lost ships

**Comment by rexxar-tc** [comment]
@GornyakManiac No, it was a deliberate decision to disable config while the server is running

**Comment by Neimoh** [comment]
- automated backups that move and compress saves then purge by age

**Comment by rexxar-tc** [comment]
@Neimoh https://github.com/sirhamsteralot/TorchBackupSystem

**Comment by Neimoh** [comment]
- stop ships on startup

**Comment by Neimoh** [comment]
that backup system doesn't compress the backups

**Comment by Neimoh** [comment]
- waypoint system

**Comment by GornyakManiac** [comment]
Inventories editing in "entities" tab

**Comment by GornyakManiac** [comment]
Grids sorting in "entitiy manager" tab

**Comment by GornyakManiac** [comment]
Command to delete unreachable underground grids inside the planets

**Comment by bigdestroyer** [comment]
Discord and ingame chat integration. Talk to online players via discord and vice versa

**Comment by ProsperCraft** [comment]
@bigdestroyer There is already a plugin for this, and it works well.

**Comment by ProsperCraft** [comment]
I would love to see some way of knowing what was causing performance issues, perhaps if the server would notify if a single grid was causing big performance drops, like a grid grinding on the ground singularly causing the sim to drop.

**Comment by smooreace** [comment]
Profiler already does that

**Comment by Murllin** [comment]
Block Enforcement. (Max per grid specifically)

**Comment by bigdestroyer** [comment]
can someone perhaps give me a guide how to enable the discord integration?

**Comment by theCalcaholic** [comment]
@bigdestroyer Have a look here: https://github.com/reactiflux/discord-irc/wiki/Creating-a-discord-bot-&-getting-a-token

**Comment by theCalcaholic** [comment]
* condition for the cleanup command that checks for a grid's distance to players.

**Comment by Varaxian** [comment]
Admins not being moderated by the block limiter would be super fantastic.

**Comment by Arstraea** [comment]
Control system for blocking respawnship farming.
 - Increase a cooltime double or specific multiplier in config when user roll-out a new respawnship
  ex)
    1st Respawnship : 15min
    2nd Respawnship : 30min
    3rd Respawnship : 1Hour
    4th Respawnship : 2Hour
    ...

 - Statistics or log that counting each user spawn his new respawnship.
  ex)
    User_A : PlanetaryLander - 4 times
    User_B : PlanetaryLander - 122 times
